src/ray/common/ray_config_def.h

```diff
@@ -298,6 +298,10 @@ RAY_CONFIG(bool, put_small_object_in_memory_store, false)
 /// pipelining task submission.
 RAY_CONFIG(uint32_t, max_tasks_in_flight_per_worker, 1)
 
+/// Enable stealing of non-actor tasks among workers that are associated with the same
+/// owner
+RAY_CONFIG(bool, work_stealing, false)
```

Comment:
What do you think about always enabling work stealing? I think this would be more robust since we won't have to check an additional config in direct_task_transport (when max_tasks_in_flight=1, enabling work stealing should have the same behavior as no work stealing).

Reply:
yes, given the results of the experiments, it looks like leaving work stealing enabled is safe and it never degrades performance! The one config that will need more careful reasoning is the default max_tasks_in_flight setting (which is currently just 1)